index.js

```diff
@@ -34,15 +40,15 @@ program
       if (results.length === 0) {
         console.log(chalk.red('No results were found, please try another phrase'));
       } else {
-        if (results.length > 3) {
-          trimRes = results.slice(0, 3);
+        if (results.length > num) {
+          trimRes = results.slice(0, num);
         } else {
           trimRes = results;
         }
         trimRes.forEach((result) => {
           if (typeof (result.definition) !== undefined) {
-            console.log('Word: ' + chalk.magenta(result.word));
-            console.log('Definition: ' + chalk.magenta(result.definition));
+            console.log('Word: ' + chalk.bold(result.word));
```

Comment:
Can you make the title words bold and colored, and return the results as default text color instead? Like we discussed last night.